examples/spatial/malaria_prevalence_PyMCv5.myst.md

```diff
@@ -0,0 +1,391 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: pymc
+  language: python
+  name: python3
+---
+
+(The prevalence of malaria in the Gambia)=
+# The prevalence of malaria in the Gambia
+
+:::{post} Aug 04, 2024 
+:tags: spatial, autoregressive, count data
```

Comment:
these are too detailed, please take a look at existing tags and re-use those, should add a GP one.

Reply:
Hey @twiecki , I had copied those tags from the notebook "Conditional Autoregressive (CAR) Models for Spatial Data". Would it make more sense if the tags were just "spatial" and "gp"?

Reply:
GP should be added. I have no strong opinion on the others.